dashboard.el

```diff
@@ -25,6 +25,14 @@
 (eval-when-compile (require 'projectile))
```

Comment:
This could be removed btw given that projectile is only needed if we have the projectile widget in the list